src/main/webapp/settings.html

```diff
@@ -35,6 +35,9 @@ <h1>Create new family</h1>
                 <button type="button" class="btn cancel" onclick="closeFamilyForm()">Close</button>
               </form>
             </div>
+            <form action="/login" method="GET">
+                <button onclick="userLogin()" name="logoutBtn">Logout</button>
```

Comment:
the userLogin function might be unnecessary if the form already makes the get request when submitted

Reply:
solved